Classes/Domain/Service/ImportService.php

```diff
@@ -86,6 +86,7 @@ public function start($identifier = null, $force = false)
         }
 
         $this->importRepository->add($this->currentImport);
+        $this->persistenceManager->persistAll();
```

Comment:
If we don't do this the importer subprocess will be executed (and fail) before the changes are persisted in the database.